operating/manage-network-policies.adoc

```diff
@@ -44,13 +44,19 @@ include::modules/delete-all-autogenerated-policies.adoc[leveloffset=+2]
 
 include::modules/policy-generation-strategy.adoc[leveloffset=+2]
 
-== Using static network policy generator
-:FeatureName: Static network policy generation
+== Using build-time network policy generator
+:FeatureName: Build-time network policy generation
 include::snippets/technology-preview.adoc[]
 
-The static network policy generator analyzes Kubernetes and {ocp} manifests and generates a set of suggested network policies in YAML format. You can apply these generated policies to your cluster. The static network policy generator uses link:https://github.com/np-guard/cluster-topology-analyzer[NP-Guard's Cluster Topology Analyzer], which discovers the network connections and creates the network policies.
+The `roxctl` command-line interface (CLI), can generate Kubernetes network policies based on Application YAML manifests.
+You can use it to develop network policies as part of the CI/CD pipeline before deploying applications on your cluster.
 
-include::modules/generate-static-network-policies.adoc[leveloffset=+2]
+Red Hat developed this feature in partnership with the developers of the link:https://np-guard.github.io/[NP-Guard project].
+First, the build-time network policy generator analyzes Kubernetes manifests in a local folder, including service manifests, configuration maps, and workload manifests such as Pod, Deployment, ReplicaSet, Job, DaemonSet, and StatefulSet.
```

Comment:
Please use backticks for the Pod, Deployment, ReplicaSet, Job, DaemonSet, and StatefulSet objects/resources.